README.md

```diff
@@ -94,6 +96,7 @@ resource "aws_cloudwatch_metric_alarm" "app_service_low_cpu" {
 - `name` - Name of the service in `CamelCase` without spaces
 - `vpc_id` - ID of VPC housing the service
 - `public_subnet_ids` - A list of public subnet IDs used to place load balancers
+- `security_group_ids` - A list of security group IDs for the load balancer (default: `[]`)
```

Comment:
Agreed that we should signal that this defaults to a list with one element inside (the `lb_security_group_id`).